api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+#This file provide the end points for interaction with RDR
+
+
+swagger: '2.0'
+
+info:
+  title: RDR
+  description: |
+    Service for RDR Interactions.
+  version: "0.1"
+#The host will get updated as per the environment it will be pmi-drc-api-test for dev and test
+host: "pmi-drc-api-test.appspot.com"
+schemes:
+  - "https"
+basePath: /rdr/v1
+
+securityDefinitions:
+  googleoauth:
+    type: oauth2
+    authorizationUrl: 'https://accounts.google.com/o/oauth2/auth'
+    flow: implicit
+    scopes:
+      openid: open id authorization
+      email: email authorization
+      profile: profile authorization
+
+security:
+  - googleoauth:
+    - openid
+    - email
+    - profile
+    - https://www.googleapis.com/auth/devstorage.full_control
+
+produces:
+  - application/json
+
+##########################################################################################
+## PATHS
+##########################################################################################
+
+paths:
+  /workbench/directory/workspaces:
+    post:
+      description: Submits one or more workspace snapshots for storage in the RDR.
+      tags:
+        - RDR
+      consumes:
+        - application/json
+      operationId: sendWorkspace
+      parameters:
+        - in: body
+          required: true
+          name: workspaces
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RDRWorkspace"
+      responses:
+        200:
+          description: successful operation
+
+  /workbench/directory/researchers:
+    post:
+      description: Submits one or more researcher snapshots for storage in the RDR.
+      tags:
+        - RDR
+      operationId: sendResearcher
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          required: true
+          name: researchers
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RDRResearcher"
+      responses:
+        200:
+          description: successful operation
+
+
+
+definitions:
+  RDRWorkspace:
+    type: object
+    properties:
+      workspaceId:
+        type: integer
+      name:
+        type: string
+      creationTime:
```

Comment:
Would it be too late to work with the RDR team to rename this field to "createdTime"? See my comments in the Liquibase changelog for why I think it would be preferred. It would be nice to have the RDR API align with our internal schema where possible.